doc: Add known issues for crypto configurations (#11882)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to :ref:`nrfxlib:crypto`.

.. rst-class:: v2-4-1 v2-4-0

NCSDK-22091: Selecting both :kconfig:option:`NORDIC_SECURITY_BACKEND` and :kconfig:option:`PSA_CORE_BUILTIN` causes a build failure
  Selecting both :kconfig:option:`NORDIC_SECURITY_BACKEND` and :kconfig:option:`PSA_CORE_BUILTIN` results in a build failure due to undefined references to different structs.

  **Workaround:** Manually define ``PSA_CORE_BUILTIN`` in the file :file:`nrf_security/configs/legacy_crypto_config.h.template`.

> .. rst-class:: v2-4-0 v2-4-1

Reviewer: ```suggestion
.. rst-class:: v2-4-1 v2-4-0
```

> NCSDK-22593: Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` causes build failure

Reviewer: ```suggestion
NCSDK-22593: Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` causes a build failure
```

>   Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` will result in a build failure due to unsatissified dependencies in check_config.h.

Reviewer 2: ```suggestion
  Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` will result in a build failure due to unsatisfied dependencies in check_config.h.
```

>   Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` will result in a build failure due to unsatissified dependencies in check_config.h.

Reviewer: ```suggestion
  Selecting :kconfig:option:`PSA_WANT_ALG_CCM` without :kconfig:option:`MBEDTLS_AES_C` results in a build failure due to unsatisfied dependencies in :file:`check_config.h`.
```

Author: I did it as well but I could commit the suggestion as is because of conflicts. Please check :)

>   **Workaround:** Manually define MBEDTLS_AES_C in the file nrf_security/configs/nrf-config.h or enable :kconfig:option:`PSA_WANT_ALG_GCM`.

Reviewer: ```suggestion
  **Workaround:** Manually define ``MBEDTLS_AES_C`` in the file :file:`nrf_security/configs/nrf-config.h` or enable :kconfig:option:`PSA_WANT_ALG_GCM`.
```
